src/test/regress/expected/multi_complex_expressions.out

```diff
@@ -445,15 +445,15 @@ DEBUG:  push down of limit count: 30
  l_partkey | o_custkey | l_quantity 
 -----------+-----------+------------
     154380 |       421 |      26.00
-    103981 |       325 |      26.00
+    138854 |        79 |      26.00
```

Comment:
Why these numbers changed? I did not understand their relation with constraints on append table.